src/main/java/seedu/address/storage/StorageManager.java

```diff
@@ -55,7 +55,7 @@ public Path getAddressBookFilePath() {
 
     @Override
     public Optional<ReadOnlyAddressBook> readAddressBook() throws DataConversionException, IOException {
-        return readAddressBook(addressBookStorage.getAddressBookFilePath());
+        return readAddressBook(addressBookStorage.getAddressBookFilePath()); //Obtaining file path
```

Comment:
Not sure if the additional comment is meant to be left in, but its fine to me

Reply:
Don't find this to be a big issue 😄